src/app/forms/custom-sections/weights/weights.component.ts

```diff
@@ -103,20 +103,23 @@ export class WeightsComponent implements OnInit, OnDestroy, OnChanges {
   }
 
   private handleVehicleTechRecordChange(changes: SimpleChanges): void {
-    if (changes['vehicleTechRecord'] && !this.ladenWeightOverride && this.form) {
-      const { currentValue, previousValue } = changes['vehicleTechRecord'];
+    const { vehicleTechRecord } = changes;
```

Comment:
Why has this been removed from the inline if?

Reply:
I added it back.